File: jashing/src/main/kotlin/com/github/avarabyeu/jashing/core/IncorrectConfigurationException.java
package com.github.avarabyeu.jashing.core;

/**
 * Incorrect configuration exception
 *
 * @author avarabyeu
 */
public class IncorrectConfigurationException extends RuntimeException {

    public IncorrectConfigurationException(String message) {
        super(message);
    }

    public IncorrectConfigurationException(String message, Throwable cause) {
        super(message, cause);
    }
}

File: jashing/src/main/kotlin/com/github/avarabyeu/jashing/core/IncorrectConfigurationException.kt
package com.github.avarabyeu.jashing.core

/**
 * Incorrect configuration exception

 * @author avarabyeu
 */
class IncorrectConfigurationException : RuntimeException {

    constructor(message: String) : super(message)

    constructor(message: String, cause: Throwable) : super(message, cause)
}
